beacon_node/lighthouse_network/gossipsub/src/topic.rs

```diff
@@ -82,6 +82,10 @@ impl TopicHash {
     pub fn as_str(&self) -> &str {
         &self.hash
     }
+
+    pub fn hash_byte_len(&self) -> usize {
+        self.hash.as_bytes().len()
+    }
```

Comment:
nit: we are only using this once  and it's a one liner, no need for a function to abstract the one liner